api/v1/routes/exercises.rb

```diff
@@ -58,6 +58,7 @@ class Exercises < Core
         require_key
 
         track_ids = [id.downcase] & ::Xapi.tracks.map(&:id)
+        halt 404, { error: "No track '%s'" % id }.to_json if track_ids.empty?
```

Comment:
Could we use https://github.com/exercism/x-api/blob/3b7b001703df8bd19d44f162aa859eedb4c8e13e/api/helpers/finders.rb#L4-L10 here instead of the two lines above?